build-static.sh

```diff
@@ -19,9 +19,8 @@ if [ -z "${PHP_EXTENSIONS}" ]; then
     export PHP_EXTENSIONS="apcu,bcmath,bz2,calendar,ctype,curl,dba,dom,exif,fileinfo,filter,gd,iconv,igbinary,intl,ldap,mbregex,mbstring,mysqli,mysqlnd,opcache,openssl,pcntl,pdo,pdo_mysql,pdo_pgsql,pdo_sqlite,pgsql,phar,posix,readline,redis,session,simplexml,sockets,sodium,sqlite3,sysvsem,tokenizer,xml,xmlreader,xmlwriter,zip,zlib" 
 fi
 
-if [ -z "${PHP_EXTENSION_LIBS}" ]; then
-    export PHP_EXTENSION_LIBS="bzip2,freetype,libavif,libjpeg,liblz4,libwebp,libzip"
-fi
+# the Brotli library must always be built as it is required by http://github.com/dunglas/caddy-cbrotli
+export PHP_EXTENSION_LIBS="${PHP_EXTENSION_LIBS},brotli,bzip2,freetype,libavif,libjpeg,liblz4,libwebp,libzip"
```

Comment:
Shouldn't we keep the `if` to allow users removing libs?

Reply:
According to my understanding, frankenPHP currently requires `brotli`, and the remaining libraries here are optional libraries that are forced to be enabled.

If these can be removed, should the brotli be retained?

Reply:
Yes, Brolti is the only one that is mandatory. I'll try to make it optional again at some point.